react/features/app/functions.any.js

```diff
@@ -26,3 +27,31 @@ export function getAppProp(stateful: Function | Object, propName: string) {
 
     return undefined;
 }
+
+/**
+ * Retrieves the default URL for the app. This can either come from a prop to
+ * the root App component or be configured in the settings.
+ *
+ * @param {Function|Object} stateful - The redux store or {@code getState}
+ * function.
+ * @returns {string} - Default URL for the app.
+ */
+export function getDefaultURL(stateful: Function | Object) {
+    const state = toState(stateful);
+    const { app } = state['features/app'];
+
+    // If the execution environment provides a Location abstraction, then
+    // this App at already at that location but it must be made aware of the
+    // fact.
+    const windowLocation = app.getWindowLocation();
```

Comment:
At the moment this `getWindowLocation` function is only used here. I'm considering replacing it with a explicit check for React Native, since we have no other use for it. Thoughts?